change/@fluentui-react-examples-2021-01-22-12-35-03-hotell-cherry-pick-16439.json

```diff
@@ -1,8 +0,0 @@
-{
-  "type": "patch",
```

Comment:
Was this intentional?

Reply:
Definitely not, always fighting against the changes 😄 Will fix.